backend/src/packages/chat-messages/chat-message.service.ts

```diff
@@ -0,0 +1,74 @@
+import { ExceptionMessage, HTTPCode, UsersError } from 'shared/build/index.js';
+
+import { type Service } from '#libs/types/types.js';
+import { userService } from '#packages/users/users.js';
+
+import { ChatMessageEntity } from './chat-message.entity.js';
+import { type ChatMessageRepository } from './chat-message.repository.js';
+import {
+  type ChatMessageCreateData,
+  type ChatMessageGetAllItemResponseDto,
+} from './libs/types/types.js';
+
+class ChatMessageService implements Service {
+  private chatMessageRepository: ChatMessageRepository;
+
+  public constructor(chatMessageRepository: ChatMessageRepository) {
+    this.chatMessageRepository = chatMessageRepository;
+  }
+
+  public find(): ReturnType<Service['find']> {
+    return Promise.resolve(null);
+  }
+
+  public async findAll(): ReturnType<Service['findAll']> {
+    return await Promise.resolve({ items: [] });
+  }
+
+  public async create(
+    payload: ChatMessageCreateData,
+  ): Promise<ChatMessageGetAllItemResponseDto> {
+    const sender = await userService.findById(payload.senderId);
+    if (!sender) {
+      throw new UsersError({
+        status: HTTPCode.NOT_FOUND,
+        message: ExceptionMessage.USER_NOT_FOUND,
+      });
+    }
+
+    const chatMessage = await this.chatMessageRepository.create(
+      ChatMessageEntity.initializeNew({
+        message: payload.message,
+        chatId: payload.chatId,
+        senderId: payload.senderId,
```

Comment:
we should take sender id from authenticated user
```suggestion
```

Reply:
Not resolved